elasticdl/python/worker/worker.py

```diff
@@ -223,6 +228,31 @@ def get_model_from_ps(self, version, method):
             model_version = max(model_version, res.model.version)
         self._model_version = model_version
 
+    def pull_embedding_vector(self, layer_name, embedding_ids):
+        """Pulls and returns embedding vectors as the order of ids."""
```

Comment:
What does "as the order of ids" mean? Do you mean "ordered by embedding id"?

Reply:
Yes, I will modify this docs.